lxd/api.go

```diff
@@ -98,6 +101,12 @@ func restServer(d *Daemon) *http.Server {
 		mux.HandleFunc("/ui", func(w http.ResponseWriter, r *http.Request) {
 			http.Redirect(w, r, "/ui/", http.StatusMovedPermanently)
 		})
+	} else {
+		mux.HandleFunc("/ui", func(w http.ResponseWriter, r *http.Request) {
+			w.Header().Set("Content-Type", "text/html")
+			w.WriteHeader(http.StatusServiceUnavailable)
+			fmt.Fprint(w, errorMessage)
+		})
```

Comment:
The `/ui` and `/ui/login` can be solved with a redirection to `/ui/` and a regular expression in the path, like this 
```suggestion
	} else {
		mux.HandleFunc("/ui", func(w http.ResponseWriter, r *http.Request) {
			http.Redirect(w, r, "/ui/", http.StatusMovedPermanently)
		})
		mux.HandleFunc("/ui/{_:.+}", func(w http.ResponseWriter, r *http.Request) {
			w.Header().Set("Content-Type", "text/html")
			w.WriteHeader(http.StatusServiceUnavailable)
			fmt.Fprint(w, errorMessage)
		})
	}
```
Not sure what should be the behavior in case the requested url is for example `/foo` from a browser and the ui is not enabled. In that case I think that we can intercept it but give a different error